app/ai_backend/test_genesis_connector.py

```
@@ -789,56 +1674,101 @@ def test_send_request_with_binary_payload(self, mock_post):
         
-        result = self.connector.send_request(binary_payload)
+        try:
+            formatted = self.connector.format_payload(large_payload)
+            self.assertIsNotNone(formatted)
+        except (MemoryError, ValueError) as e:
+            # Should handle gracefully
+            self.assertIsInstance(e, (MemoryError, ValueError))
+
+    def test_malformed_json_responses(self):
+        """Test handling of various malformed JSON responses."""
+        malformed_responses = [
```

Ruff (0.11.9)

Local variable `malformed_responses` is assigned to but never used

Remove assignment to unused variable `malformed_responses`

(F841)